src/Umbraco.Web.Common/Attributes/AngularJsonOnlyConfigurationAttribute.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Buffers;
+using System.Collections.Generic;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Umbraco.Web.Common.Formatters;
+
+namespace Umbraco.Web.Common.Attributes
```

Comment:
I agree, not perfect, but pretty aligned with the MVC pattern, "Views", "Controllers", ...

The issue here is it is not all filters, or related to MVC.

I will give it a try, to find better name or locations of the files

Reply:
Cool, though some argue against the MVC pattern of "Views", "Controllers", etc... .too :) which is why there's feature folders https://codingsight.com/asp-net-core-mvc-feature-folders/ 

I'd also prefer we don't just use "Controllers" and instead break that up into it's features. Could be 'BackOffice', or like we do currently for back office editors, those all exist in `Editors`.

I guess so long as we have a few tasks later on about re-adjusting all namespaces we don't have to 'solve' this today but I just worry about leaving it until later because based on past predictions these cleanup tasks won't actually get done.